release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1598,6 +1598,156 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, when you deleted backend service resources on {gcp-first}, some resources to be deleted were not found. For example, the associated forwarding rules, health checks, and firewall rules were not deleted. With this release, the installer tries to find the backend service by name first, then searches for forwarding rules, health checks, and firewall rules before it determines if those results match a backend service. The algorithm for associating resources is reversed and the appropriate resources are deleted. There are no leaked backend service resources and the issue is resolved. When you destroy a private cluster, the forwarding rules, backend services, health checks, and firewall rules created by the Ingress Operator are not deleted. (link https://issues.redhat.com/browse/OCPBUGS-48611[*OCPBUGS-48611*])
+
+* Previously, when you attempted to set the policy for a {gcp-first} service account, the API reported a `400: Bad Request` validation error. When you create a service account, it might take up to 60 seconds for the account to become active, and this causes the validation error. If this error occurs, create a service account with a true exponential backoff that lasts at least 60 seconds. (link:https://issues.redhat.com/browse/OCPBUGS-48187[*OCPBUGS-48187*])
+	
+* Previously, {product-title} was not compliant with PCI-DSS/BAFIN regulations. With this release, the cross-tenant object replication in {azure-first} is unavailable. Consequently, the chance of unauthorized data access is reduced and the strict adherence to data governance policies is ensured. (link:https://issues.redhat.com/browse/OCPBUGS-48118[*OCPBUGS-48118*])
+	
+* Previously, when you installed {product-title} on {aws-first} and specified an edge machine pool without an instance type, in some instances it caused the edge node to fail. With this release, if you specify an edge machine pool without an instance type you must use the permission `ec2:DescribeInstanceTypeOfferings`. The permission derives the correct instance type available, based on the {aws-short} Local Zones or Wavelength Zones locations used. (link:https://issues.redhat.com/browse/OCPBUGS-47502[*OCPBUGS-47502*])
+
+* Previously, when the API server disconnected temporarily, the command `oc adm node-image monitor` reported an end of file (EOF) error. With this release, when the API server disconnects temporarily, the monitor command does not fail. (link:https://issues.redhat.com/browse/OCPBUGS-46391[*OCPBUGS-46391*])
+//------ page 1
+* Previously, when you specified the `HostedZoneRole` permission in the `install-config.yaml` file while creating a shared Virtual Private Cloud (VPC), you also had to specify the `sts:AssumeRole` permission. Otherwise, it caused an error. With this release, if you specify the `HostedZoneRole` permission the installer validates that the `sts:AssumeRole` permission is present.
```

Comment:
```suggestion
* Previously, when you specified the `HostedZoneRole` permission in the `install-config.yaml` file while creating a shared Virtual Private Cloud (VPC), you also had to specify the `sts:AssumeRole` permission. Otherwise, it caused an error. With this release, if you specify the `HostedZoneRole` permission the installation program validates that the `sts:AssumeRole` permission is present.
```